modules/light-clients/07-tendermint/types/update_test.go

```diff
@@ -100,6 +105,70 @@ func (suite *TendermintTestSuite) TestCheckHeaderAndUpdateState() {
 			},
 			expPass: true,
 		},
+		{
+			name: "successful update with identical header to a previous update",
```

Comment:
what is this testing?

Reply:
It's testing that submitting a header for which the consensus state already exists passes (no-op)